recursion.md

```diff
@@ -0,0 +1,3 @@
+##RECURSION
+  *a function which called itself
+  *when it reaches the base case,it stops
```

Comment:
@Legion112 if you have something to add you can

Reply:
I don't have